security/tailscale/src/opnsense/www/js/widgets/Metadata/Tailscale.xml

```diff
@@ -0,0 +1,25 @@
+<metadata>
+    <Tailscale>
+        <filename>Tailscale.js</filename>
+        <endpoints>
+            <endpoint>/api/tailscale/service/status</endpoint>
+            <endpoint>/api/tailscale/status/status</endpoint>
```

Comment:
can you check with "make lint-acl".. relatively sure it will complain

Reply:
root@OPNsense:/usr/plugins/security/tailscale # make lint-acl
Did not find ACLBIN, please provide a core repository
*** Error code 1

Reply:
it did, fixed.